src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -634,7 +634,7 @@ def is_track2(client_class):
     """
     from inspect import getfullargspec as get_arg_spec
     args = get_arg_spec(client_class.__init__).args
-    return "credential" in args
+    return "credential" in args or hasattr(client_class, "_send_request")
```

Comment:
@msyyc I actually shouldn't have hid the signature yesterday, since that hides the signature for end users. I ended up changing it in the PR before you released yesterday. However this does make sense. For our new DPG generations, we expose send_request as public, so we can maybe add that check as well. But I think I did mess up yesterday by hiding the init signature, so I think it's ok for the CLI code to stay as is

Reply:
the PR is discussed about in e-mail: RE: Testing candidates azure-core on CLI testsuite